static/app/utils/replays/hooks/useExtractedCrumbHtml.tsx

```diff
@@ -135,11 +137,21 @@ class BreadcrumbReferencesPlugin {
         // @ts-expect-error
         const html = node?.outerHTML || node?.textContent || '';
 
-        this.activities.push({
-          crumb,
-          html,
-          timestamp: nextTimestamp,
-        });
+        // Limit document node depth to 2
+        let truncated = removeNodesAtLevel(html, 2);
+        // If still very long and/or removeNodesAtLevel failed, truncate
+        if (truncated.length > 2000) {
+          truncated = truncated.substring(0, 2000);
+        }
+
+        if (truncated) {
+          this.activities.push({
+            crumb,
+            html: truncated,
+            timestamp: nextTimestamp,
+          });
+        }
```

Comment:
```suggestion
        this.activities.push({
          crumb,
          html: truncated,
          timestamp: nextTimestamp,
        });
```

Reply:
I thought we wanted to not include empty code blocks?

Reply:
oh yeah. i guess i was reading it funny. i was reading it like "if we got a string back after calling removeNodesAtLevel and possibly calling  substring..." and i was like, sure we'll get something back.